Automattic/jetpack
# Social: Update screenshot

## Proposed changes:

* Update the screenshot to reflect the new toggles

### Other information:


## Jetpack product discussion


p1695905290749399/1695902889.372849-slack-C02JJ910CNL
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* Just see the screenshot

Current: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/.w.org-assets/screenshot-3.png
New https://github.com/Automattic/jetpack/blob/update/social-screenshot/projects/plugins/social/.w.org-assets/screenshot-3.png
